react/templates/src/setupProxy.js

```diff
@@ -0,0 +1,17 @@
+// 'React-scripts start' sets up a development server that automatically
+// registers files with this name. This proxy file sets up a CORS header
+// for manifest.json, allowing sign in via Blockstack without using
+// the webpack configuration file that create-react-app has configured 
+// and hidden.
```

Comment:
Great comment! A link to their docs about this proxy setup would also be useful.

Reply:
Will do! I'll update the PR with the other comments incorporated too